deploy/list-connectors.ts

```diff
@@ -0,0 +1,78 @@
+import { glob } from "tinyglobby";
+import { join } from "node:path";
+import Module from 'node:module';
+import fs from 'fs-extra'
```

Comment:
I use fs-extra because 
- this script is package so we don't have to care about bundle size
- fs-extra is convenient and one of my fav tool (improves readability)